browser/src/Editor/NeovimEditor/WelcomeBufferLayer.tsx

```diff
@@ -553,74 +590,74 @@ export interface IWelcomeCommandsViewProps extends Partial<WelcomeViewProps> {
-                        selected={isSelected(commands.openWorkspaceFolder.command)}
+                        selected={this.isSelected(commands.openWorkspaceFolder.command)}
                     />
                     <WelcomeButton
                         title="Command Palette"
-                        onClick={() => executeCommand(commands.commandPalette.command)}
+                        onClick={() => commands.quickOpenShow.execute()}
                         description="Control + Shift + P"
-                        command={commands.commandPalette.command}
-                        selected={isSelected(commands.commandPalette.command)}
+                        command={commands.quickOpenShow.command}
+                        selected={this.isSelected(commands.quickOpenShow.command)}
                     />
                     <WelcomeButton
                         title="Vim Ex Commands"
                         description=":"
                         command="editor.openExCommands"
-                        onClick={() => executeCommand(commands.commandline.command)}
-                        selected={isSelected(commands.commandline.command)}
+                        onClick={() => commands.commandline.execute()}
+                        selected={this.isSelected(commands.commandline.command)}
                     />
                 </AnimatedContainer>
                 <AnimatedContainer duration="0.25s">
                     <SectionHeader>Learn</SectionHeader>
                     <WelcomeButton
                         title="Tutor"
-                        onClick={() => executeCommand(commands.openTutor.command)}
+                        onClick={() => commands.openTutor.execute()}
                         description="Learn modal editing with an interactive tutorial."
                         command={commands.openTutor.command}
-                        selected={isSelected(commands.openTutor.command)}
+                        selected={this.isSelected(commands.openTutor.command)}
                     />
                     <WelcomeButton
                         title="Documentation"
-                        onClick={() => executeCommand(commands.openDocs.command)}
+                        onClick={() => commands.openDocs.execute()}
                         description="Discover what Oni can do for you."
                         command={commands.openDocs.command}
-                        selected={isSelected(commands.openDocs.command)}
+                        selected={this.isSelected(commands.openDocs.command)}
                     />
                 </AnimatedContainer>
                 <AnimatedContainer duration="0.25s">
                     <SectionHeader>Customize</SectionHeader>
                     <WelcomeButton
                         title="Configure"
-                        onClick={() => executeCommand(commands.openConfig.command)}
+                        onClick={() => commands.openConfig.execute}
```

Comment:
This doesn't seem to work for me?

From checking the menu, we use `"oni.config.openConfigJs"` there, would that work here? Its also possible something is going weird in my setup.

Reply:
woops I left off the calling parens on that function will push a fix 👍